### DISCUSSION TITLE
Trackers for movies and shows

### DISCUSSION THREAD

**Discussion by Nintenstation** [aniyomiorg/aniyomi#1350] [Ideas]
We have trackers for anime... But what about trackers like letterboxd or IMDB, that can track non-anime shows? Since we have non-anime extensions, like fmovies and sflix, I think we should have some trackers like that if it can be implemented.

**Comment by sayedhfatimi** [comment]
like trakt? since we already have simkl

**Comment by Azmarine13** [comment]
You are looking for a Tracking site like **SIMKL.com for Movies, TV Shows, Anime**
Also has browser extension: https://simkl.com/apps/chrome/
